cli/src/commands/util/gc.rs

```diff
@@ -24,6 +24,9 @@ use crate::command_error::CommandError;
 use crate::ui::Ui;
 
 /// Run backend-dependent garbage collection.
+///
+/// To garbage-collect old operations and the commits/objects referenced by
+/// then, run `jj op abandon ..<some old operation>` before `jj util gc`.
```

Comment:
I'm now thinking of adding something like:

> At present, jj's garbage collection is only partially effective. Commits in the evolution log are not yet garbage collected.

WDYT?

Reply:
Or maybe "If a commit is preserved by the garbage collection, objects reachable from its evolution log cannot be garbage collected."

Reply:
I prefer your second proposal.

Reply:
I think I'll merge this as is and make another tiny PR, to make the discussion a bit more orderly.